File: 200611/주식가격.py
def solution(prices):
    answer = []
    for i in range(len(prices) - 1):
        cnt = 0
        for j in range(i, len(prices)):
            if prices[j] < prices[i]:
                cnt+=1
                break
            cnt+=1
        answer.append(cnt-1)
    answer.append(0)
    
    return answer

File: 200611/주식가격.cs
using System;

public class Solution 
{
    public int[] solution(int[] prices) 
    {
        int[] answer = new int[prices.Length];
             
        for(int i = 0; i < prices.Length; i++) 
        {
            for(int j = i + 1; j < prices.Length; j++) 
            {
                if(prices[j] < prices[i]) 
                {
                    answer[i]++;
                    break;
                }
                answer[i]++;
            }
        }
        
        return answer;
    }
}